plotly/plotly/plotly.py

```diff
@@ -33,6 +33,7 @@
 from plotly.session import (sign_in, update_session_plot_options,
                             get_session_plot_options, get_session_credentials,
                             get_session_config)
+from plotly.grid_objs import grid_objs
```

Comment:
This import is what I was asking you about. I don't know if `plotly.plotly` can depend on the `grid_objs` module, but that's the direction I took for getting `get_grid` to output an actual GRID.